report/04_implementation/02_go_server.md

```diff
@@ -0,0 +1,27 @@
+## Go backend
+
+*Golang* is an open source programming language designed by Google developers in 2009[1].
+Because of it's simplicity and readability it is a perfect match for the programming 
+language choice on the server side of the application.
```

Comment:
```suggestion
Because it was developed for distributed systems, it's simplicity and readability
it is a perfect match for the programming language choice on the server side of the application.
```